includes/data.php

```diff
@@ -1653,6 +1740,16 @@ function pods_evaluate_tag( $tag, $sanitize = false, $fallback = null ) {
 		$value = pods_v( $tag[0], 'get', null );
 	} elseif ( 2 === count( $tag ) ) {
 		$value = pods_v( $tag[1], $tag[0], null );
+	} else {
+		// Some magic tags support traversal.
+		$value = pods_v( array_slice( $tag, 1 ), $tag[0], null );
+	}
+
+	if ( $helper ) {
+		if ( ! $pod ) {
+			$pod   = pods();
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 3 spaces

<sub>hash: 8b97d8c2cae78aae0bd92820cb56a996</sub>